libs/shared/src/containers/TransactionList/TransactionLine/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styles from './styles.module.less';
+import { Grid } from 'semantic-ui-react';
+
+type VisualProps={
+  key: number,
+  locale: string,
+
+  yearToDisplay: number,
+  monthTodisplay: string,
+  dayToDisplay: number,
+
+  imgForLine: string,
+  contentForComment: JSX.Element,
+  addressComment: string,
+  descForComment: JSX.Element,
+
+  classForMoneyCell: string,
+  changeCad: number,
+  timeToDisplay: string,
+
+  balanceCad: number
+
+};
+                
+export const TransactionLine = (props:VisualProps) => {
```

Comment:
Spend some time debating if I should have split that and I decided that maybe it was too much.

Reply:
Seems fine to me?